app/src/main/admin/components/AdminApp.tsx

```diff
@@ -1,26 +1,26 @@
 import * as React from "react";
+import { connect } from 'react-redux';
+
 import { notificationStore } from "../../shared/stores/NotificationStore";
 import { connectToStores } from "../../shared/alt";
 import { ErrorLog } from "../../shared/components/ErrorLog/ErrorLog";
 import { AdminRouter } from "./AdminRouter";
-import { adminAuthStore } from "../stores/AdminAuthStore";
 import { NotificationArea } from "../../shared/components/NotificationArea/NotificationArea";
 
 export interface AdminAppProps {
     errors: string[];
     infos: string[];
-    loggedIn: boolean;
+    loggedIn?: boolean;
```

Comment:
Maybe a nicer way of doing this is to leave `loggedIn` as a non-nullable property, but chance the type signature of `getPropsFromStores` to be `Partial<AdminAppProps>`?